internal/overlord/servstate/manager.go

```diff
@@ -300,6 +300,24 @@ func (m *ServiceManager) Services(names []string) ([]*ServiceInfo, error) {
 	return services, nil
 }
 
+// MaximumStopRunningDuration returns the worst case duration that will have to
+// be waited for to have all services in this manager stopped.
+func (m *ServiceManager) MaximumStopRunningDuration() time.Duration {
+	m.servicesLock.Lock()
+	defer m.servicesLock.Unlock()
+
+	maxDuration := defaultKillDelay
+	for _, svc := range m.services {
+		if svc.killDelay() > maxDuration {
+			maxDuration = svc.killDelay()
+		}
+	}
+
+	// We add a millisecond on to the maximum here to allow for signals to be
```

Comment:
Nit: it's 100 milliseconds, so comment doesn't match code. What about just commenting "We add a little..."